microsoft/testsuites/vm_extensions/azsecpack.py

```diff
@@ -0,0 +1,287 @@
+            assert_that(service.check_service_status(mdsd_service)).is_equal_to(True)
+            log.info(f"{mdsd_service} is running successfully")
+
+    def _check_azsec_services_status(self, node: Node, log: Logger) -> None:
+        service = node.tools[Service]
+        azsec_services = ["azsecd", "azsecmond", "auoms"]
+        for azsec_service in azsec_services:
+            assert_that(service.check_service_status(azsec_service)).is_equal_to(True)
+            log.info(f"{azsec_service} is running successfully")
+
+    def _check_azsecd_status(self, node: Node, log: Logger) -> None:
+        azsecd = node.tools[Azsecd]
+        output = azsecd.run(
+            parameters="status",
+            sudo=True,
+            expected_exit_code=0,
+            expected_exit_code_failure_message="fail to run azsecd status",
+        ).stdout
+        assert_that(output.find("Enabled(true), MdsdTenantStatus(running)") >= 0)
+        count = output.find(
+            "Path(/var/run/mdsd/asa/default_djson.socket)\n"
+            "IsAvailableToConnect(true)"
+        )
+        assert_that(count >= 0)
+        count = output.find(
+            "Path(/var/run/azsecmon/azsecmond.socket)\nIsAvailableToConnect(true)"
+        )
+        assert_that(count >= 0)
+
+    def _check_journalctl_logs(self, node: Node, log: Logger) -> None:
+        journalctl = node.tools[Journalctl]
+        output = journalctl.logs_for_unit("azsecd", sudo=True)
+        assert_that(output.find("Connected to mdsd") >= 0)
+        log.info("Azsecd connection to mdsd is successful")
+        assert_that(output.find("Scan 'certsinuse' completed") >= 0)
+        log.info("Scan 'certsinuse' completed")
+        assert_that(output.find("Scan 'heartbeat' completed") >= 0)
+        log.info("Scan 'heartbeat' completed")
+
+        output = journalctl.logs_for_unit("azsecmond", sudo=True)
+        assert_that(output.find("Connected to mdsd") >= 0)
+        log.info("Azsecmond connection to mdsd is successful")
+
+        output = journalctl.logs_for_unit("auoms", sudo=True)
+        assert_that(output.find("Output(mdsd): Connected") >= 0)
+        log.info("Auoms connection to mdsd is successful")
+        assert_that(output.find("Output(azsecmond): Connected") >= 0)
+        log.info("Auoms connection to azsecmond is successful")
+
+    def _check_mdsd_qos_logs(self, node: Node, log: Logger) -> None:
+        cat = node.tools[Cat]
+        mdsd_logs = cat.read("/var/log/mdsd/asa.qos", sudo=True)
+        loop_count = 0
+        while mdsd_logs == "" and loop_count < 20:
+            mdsd_logs = cat.read("/var/log/mdsd/asa.qos", sudo=True)
+            loop_count += 1
+            time.sleep(5)
+
+        assert_that(
+            mdsd_logs.find("MaRunTransmitBondNotification,LinuxAsmHeartbeat") >= 0
+        )
+        log.info("LinuxAsmHeartbeat is transmitted to Geneva")
+        assert_that(mdsd_logs.find("MaRunTransmitBondNotification,LinuxAsmAudit") >= 0)
```

Comment:
It's not a right assertion, it doesn't raise error, if the condition is not met. And there are is greater than assertion method, don't write expression in `assert_that` directly.